R/object-builder.R

```diff
@@ -243,7 +243,7 @@ upgrade_fcm <- function(x) {
     if (!is_pre2(x)) return(x)
     attrs <- attributes(x)
     build_fcm(
-        x, colnames(x),
+        x, rownames(x), colnames(x),
```

Comment:
Does it make sense to have a non-symmetric fcm? I can't think of how it does.

Reply:
Non-square FCM can exists, so just to avoid error
```
> fcmt[1,1:10]
Feature co-occurrence matrix of: 1 by 10 feature.
        features
features fellow - citizens  of the senate and house representatives :
  fellow      3 6       16 224 361      3 146     6               6 4
```